<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LegendTab</class>
 <widget class="QWidget" name="LegendTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>653</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_title"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="_autoContents">
     <property name="text">
      <string>Use Automatic plot based Contents</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_contentGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel4_4_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Available objects:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_availableRelationList</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2" colspan="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>71</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="TextLabel3_3_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Displayed objects:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_displayedRelationList</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" rowspan="5">
       <widget class="QListWidget" name="_availableRelationList"/>
      </item>
      <item row="1" column="4" rowspan="5">
       <widget class="QListWidget" name="_displayedRelationList"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="_up">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Raise in plot order: Alt+up</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Alt+Up</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="_remove">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Remove: Alt+r</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="_add">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select: Alt+s</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Alt+S, Alt+F, Alt+F</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="_down">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Lower in plot order: Alt+down</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="shortcut">
         <string>Alt+Down</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>128</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="_topLabelGroup">
     <property name="title">
      <string>Legend Font Settings</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="_Label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Font Si&amp;ze Offset</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>_fontSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="_fontSize">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-25.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="_Label_12">
        <property name="text">
         <string>&amp;Font Family</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>_family</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QFontComboBox" name="_family"/>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="_bold">
        <property name="text">
         <string>&amp;Bold</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="_underline">
        <property name="text">
         <string>&amp;Underline</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="_italic">
        <property name="text">
         <string>&amp;Italics</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Layout</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="_displayVertically">
        <property name="text">
         <string>Display &amp;Vertically</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_title</tabstop>
  <tabstop>_autoContents</tabstop>
  <tabstop>_availableRelationList</tabstop>
  <tabstop>_displayedRelationList</tabstop>
  <tabstop>_up</tabstop>
  <tabstop>_remove</tabstop>
  <tabstop>_add</tabstop>
  <tabstop>_down</tabstop>
  <tabstop>_fontSize</tabstop>
  <tabstop>_bold</tabstop>
  <tabstop>_underline</tabstop>
  <tabstop>_italic</tabstop>
  <tabstop>_family</tabstop>
  <tabstop>_displayVertically</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
